internal/provider/resource_password.go

```diff
@@ -72,13 +72,32 @@ func createPassword(ctx context.Context, d *schema.ResourceData, meta interface{
 }
 
 func importPasswordFunc(ctx context.Context, d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	for k, v := range passwordSchemaV2() {
+		if v.Default == nil {
+			continue
+		}
+		if err := d.Set(k, v.Default); err != nil {
+			return nil, fmt.Errorf("error setting %s: %w", k, err)
+		}
```

Comment:
On second thought, I think there might be a minor issue here.

Before setting the value, the import should check if the configuration in the code base has been set with a value. Otherwise we risk setting the state to Default value, even though the user has indeed provided a value.

Reply:
@wakeful, I've discussed this with @detro and other members of the team and unfortunately, it's not possible to check the configuration from within a [StateContextFunc](https://github.com/hashicorp/terraform-plugin-sdk/blob/main/helper/schema/resource_importer.go#L50) (i.e., `importPasswordFunc`) as the `ResourceData` only has the `ID` set and there is no way to access the attributes and their associated values that have been defined in the TF config. 

Unfortunately, we can't issue a warning during the running of `terraform import` to alert practitioners that attributes defined within the config will be ignored as it is not currently possible to [access diagnostics during an import](https://github.com/hashicorp/terraform-plugin-sdk/issues/523).

Consequently, it seems the best we can do is to document the behaviour and highlight that practitioners can also use `ignore_changes` to opt out of plan changes.